packages/strapi-plugin-graphql/test/graphqlCrud.test.e2e.js

```diff
@@ -179,7 +179,7 @@ describe('Test Graphql API End to End', () => {
       });
     });
 
-    test('List posts with `created_by` and `updated_by`', async () => {
+    test.skip('List posts with `created_by` and `updated_by`', async () => {
```

Comment:
Why is it skipped?

Reply:
Right now, we don't have any tool to test the `populateCreatorFields: true` option on the models.
This test will be reimplemented when we'll rework the e2e test architecture.